docs/attributes-registry/process.md

```diff
@@ -34,4 +36,32 @@
 | `process.vpid` | int | Virtual process identifier. [1] | `12` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
 **[1]:** The process ID within a PID namespace. This is not necessarily unique across all processes on the host but it is unique within the process namespace that the process exists within.
+
+`process.context_switch_type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `voluntary` | voluntary | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `involuntary` | involuntary | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+`process.paging.fault_type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `major` | major | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `minor` | minor | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 <!-- endsemconv -->
+
+<!-- semconv registry.process.cpu(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  | Stability |
+|---|---|---|---|---|
+| `process.cpu.state` | string | The CPU state for this data point. A process SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels. | `system` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Shall we honor the signal-generic rule here?
This was discussed at https://github.com/open-telemetry/semantic-conventions/pull/681/files#r1476037740 and we have followed that so far in other namespaces as well like in `system` and `container`.

Reply:
we have discussed it here https://github.com/open-telemetry/semantic-conventions/pull/681/files#r1484331195
with resolution to leave as is. Do you think we need to rewrite it?

Reply:
I wouldn't consider this as a clear and future-proof resolution 🤔 .
I would prefer being consistent for now and go with the overwrite approach like we did in `system` and `container`. 
That will also bring us less friction at https://github.com/open-telemetry/semantic-conventions/issues/840.

Reply:
Yeah I'd prefer if we keep things in the registry signal-agnostic, since the whole point of it is sharing the attribute across signals.

Reply:
I have removed description from global registry and added to specific metrics usage